core/dbt/tests/fixtures/project.py

```diff
@@ -197,7 +213,12 @@ def write_project_files(project_root, dir_name, file_dict):
 # Write files out from file_dict. Can be nested directories...
 def write_project_files_recursively(path, file_dict):
     for name, value in file_dict.items():
-        if name.endswith(".sql") or name.endswith(".csv") or name.endswith(".md"):
+        if (
+            name.endswith(".sql")
+            or name.endswith(".csv")
+            or name.endswith(".md")
+            or name.endswith(".SQL")
```

Comment:
Some tests require this

Reply:
I don't think we support SQL (as opposed to sql). We don't read csv files in models directories.

Reply:
This is needed for a [test](https://github.com/dbt-labs/dbt-core/blob/main/test/integration/008_schema_tests_tests/case-sensitive-models/uppercase.SQL). We can also remove that test if we don't want to do this

Reply:
Which tests in the old integration tests? I know that the old integration test framework did that, but it was because they had directories for multiple projects in the same test directory, and we don't need that anymore.

Reply:
Ooops. Commented on wrong comment.

Reply:
Please remove the name.endswith(".csv") and the name.endswith(".SQL"). I'd rather not clutter up the code with special stuff for a one time test. You can achieve the same thing by using the "write_file" utility in dbt.tests.util, something like: write_file(file_contents, project.project_root, "models", "something.SQL")

Reply:
Removed `.SQL`, but we can't remove `.csv` as it is being used in seeds